src/prerequisites/forms.py

```diff
@@ -47,15 +47,15 @@ def hardcoded_prereq_model_choice():
 class PrereqFormInline(FutureModelForm):
     """This form class is intended to be used in an inline formset"""
 
-    prereq_object = CachedContentObjectChoiceField(
+    prereq_object = ContentObjectChoiceField(
         queryset=QuerySetSequence(*[klass.objects.all() for klass in hardcoded_prereq_model_choice()]),
-        widget=Select2Widget,
+        widget=CustomContentObjectSelect2Widget(),
```

Comment:
can we make this the default widget for a ContentObjectChoiceField?

Reply:
see my comments bellow